libopenage/gamestate/gamestate.h

```diff
@@ -0,0 +1,111 @@
+// Copyright 2017-2017 the openage authors. See copying.md for legal info.
+
+#pragma once
```

Comment:
we added this file as a prototype but we should remove this from the pull request to then work on its creation in #964